src/pages/index.js

```diff
@@ -49,35 +65,115 @@ function handleImageClick(data) {
 }
 
 function handleEditSumbit(obj) {
+  popupEdit.renderLoading(true);
   userInfo.setUserInfo(obj);
-  popupEdit.close();
-  formEditValidator.disableButton();
+  api
+    .updateCurrentUser(
+      JSON.stringify({
+        name: obj.name,
+        about: obj.about,
+      })
+    )
+    .then(() => {
+      popupEdit.close();
+      popupEdit.renderLoading(false);
+      formEditValidator.disableButton();
+    });
 }
 
 function handleAddSumbit(obj) {
-  sectionCard.addItem(createCard(obj));
-  popupAdd.close();
-  popupAdd.form.reset();
-  formAddValidator.disableButton();
+  popupAdd.renderLoading(true);
+  api
+    .postCard(
+      JSON.stringify({
+        name: obj.name,
+        link: obj.link,
+      })
+    )
+    .then((data) => {
+      sectionCard.addItem(data, "prepend");
+    })
+    .then(() => {
+      setTimeout(10000);
+      popupAdd.close();
+      popupAdd.renderLoading(false);
+      popupAdd.form.reset();
+      formAddValidator.disableButton();
+    });
+}
+
+function handleDeleteClick(thisCard) {
+  popupDelete.getCardInfo(thisCard);
+  popupDelete.open();
+}
+
+function handleDeleteSubmit(thisCard) {
+  api.deleteCard(thisCard._id);
+  thisCard.deleteCard();
+  popupDelete.close();
+}
+
+function handleAvatarSubmit(obj) {
+  popupAvatar.renderLoading(true);
+  avatar.src = obj.link;
+  api
+    .updateCurrentAvatar(
+      JSON.stringify({
+        avatar: obj.link,
+      })
+    )
+    .then(() => {
+      popupAvatar.close();
+      popupAvatar.renderLoading(false);
+      popupAvatar.form.reset();
+    });
+}
+
+//Check state of isLiked in the server and toggle like state in the API
+function toggleServerLike(thisCard) {
+  if (!thisCard._isLiked) {
+    api.putLike(thisCard._id);
+  } else {
+    api.deleteLike(thisCard._id);
+  }
+  thisCard._isLiked = !thisCard._isLiked;
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.